examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -78,54 +88,36 @@ async function start(): Promise<void> {
     let id: string;
 
     // Get or create the document depending if we are running through the create new flow
-    const createNew = !location.hash;
+    const createNew = location.hash.length === 0;
```

Comment:
maybe this should be <= 1

because `site.url/#` would return a hash length of 1

Reply:
Either one is fine -- `site.url/#` still gives a hash length of 0.